modules/investigating-etcd-installation-issues.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-installations.adoc
+
+[id="investigating-etcd-installation-issues_{context}"]
+= Investigating etcd installation issues
+
+If you experience etcd issues during installation, you can check etcd Pod status and collect etcd Pod logs. You can also verify etcd DNS records and check DNS availability on master nodes.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+* You have installed the OpenShift CLI (`oc`).
+* You have SSH access to your hosts.
+* You have the fully qualified domain names of the master nodes.
+
+.Procedure
+
+. Check the status of etcd Pods.
+.. Review the status of Pods in the `openshift-etcd` namespace:
++
+----
+$ oc get pods -n openshift-etcd
+----
++
+.. Review the status of Pods in the `openshift-etcd-operator` namespace:
++
+----
+oc get pods -n openshift-etcd-operator
+----
+
+. If any of the Pods listed by the previous commands are not showing a `Running` or a `Completed` status, gather diagnostic information for the Pod.
+.. Review events for the Pod:
++
+----
+$ oc describe pod/<pod_name> -n <namespace>
+----
++
+.. Inspect the Pod's logs:
++
+----
+$ oc logs pod/<pod_name> -n <namespace>
+----
++
+.. If the Pod has more than one container, the preceding command will create an error, and the container names will be provided in the error message. Inspect logs for each container:
++
+----
+$ oc logs pod/<pod_name> -c <container_name> -n <namespace>
+----
+
+. If the API is not functional, review etcd Pod and container logs on each master node by using SSH instead. Replace `<master-node>.<cluster_name>.<base_domain>` with appropriate values.
+.. List etcd Pods on each master node:
++
+----
+$ ssh core@<master-node>.<cluster_name>.<base_domain> sudo crictl pods --name=etcd-
+----
++
+.. For any Pods not showing `Ready` status, inspect Pod status in detail. Replace `<pod_id>` with the Pod's ID listed in the output of the preceding command:
++
+----
+$ ssh core@<master-node>.<cluster_name>.<base_domain> sudo crictl inspectp <pod_id>
+----
++
+.. List containers related to a Pod:
++
+// TODO: Once https://bugzilla.redhat.com/show_bug.cgi?id=1858239 has been resolved, the following command can replace the `grep` alternative below:
```

Comment:
It seems like the comments here are causing a formatting issue that shows up in the preview build starting at, "For any containers not showing Ready status..."

Reply:
Thanks Bob for highlighting this. I have now added a plus sign after the comments, which seems to have resolved the formatting issue.